firefox-ios/firefox-ios-tests/Tests/XCUITests/LoginsTests.swift

```diff
@@ -59,7 +59,7 @@ class LoginTest: BaseTestCase {
         navigator.goto(LoginsSettings)
         unlockLoginsView()
         mozWaitForElementToExist(app.tables["Login List"])
-        XCTAssertTrue(app.searchFields["Filter"].exists)
+        XCTAssertTrue(app.searchFields["Search passwords"].exists)
```

Comment:
@isabelrios The changes in `LoginTests` seems not related to the XCode 15.1 upgrade, but I sneaked them in. 🙄

Reply:
https://github.com/mozilla-mobile/firefox-ios/commit/397bea560a242f8824b11a2dbfe2187b3231f8ad